sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_deployment/online/request_schema.py

```diff
@@ -0,0 +1,27 @@
+# ---------------------------------------------------------
```

Comment:
Let's add some tests, I am not able see any for ths feature all together

Reply:
This feature is currently not accessible to SDK users. The only implementation is cli and UI for now. Tests coverage exists in the cli.

Reply:
Maybe I am not following it, it should not matter if this is for SDK users or not, the test cases should be there for code coverage. We need to make sure someone else's change does not break us, and there should be some basic tests in SDK, e.g. make sure schema loads successfully and entities are formed. I would be happy to share some more details, feel free to setup time.

Reply:
have not been able to get my sdk test situation set up. I have this[ work item](https://msdata.visualstudio.com/Vienna/_workitems/edit/2166786) to add the sdk tests with the cli tests as soon as I get my set up unblocked